.gitignore

```diff
@@ -79,12 +82,6 @@ Thumbs.db
 .buildpath
 *.un~
 launch.json
-# ignore plugins directory by default except LS plugin
```

Comment:
This was a duplicated entry in the `.gitignore` file. You will find the same code [here](https://github.com/LimeSurvey/LimeSurvey/pull/945/files#diff-a084b794bc0759e7a6b77810e01874f2L29)